spec.md

```diff
@@ -2041,6 +2096,51 @@ Condition | gRPC Code | Description | Recovery Behavior
 | Call not implemented | 12 UNIMPLEMENTED | NodeGetInfo call is not implemented by the plugin or disabled in the Plugin's current mode of operation. | Caller MUST NOT retry. Caller MAY call `ControllerGetCapabilities` or `NodeGetCapabilities` to discover Plugin capabilities. |
 
 
+#### `NodeExpandVolume`
+
+A Node Plugin MUST implement this RPC call if it has `EXPAND_VOLUME` node capability.
+This RPC call allows CO to expand volume on the node.
+
+`NodeExpandVolume` MUST be called after `ControllerExpandVolume` if volume has `EXPAND_VOLUME` controller capability.
+Also `NodeExpandVolume` MUST be called only on a volume which is already published by controller if controller has `PUBLISH_UNPUBLISH_VOLUME` capability.
```

Comment:
ditto .. what about cases when that capability isn't supported?

Reply:
I have tried to expand the case of when the `NodeExpandVolume` can be called based on capabilities defines on the node and controller. PTAL